components/chart/types/column.md

```diff
@@ -67,6 +78,47 @@ The color of a series is controlled through the `Color` property that can take a
 
 @[template](/_contentTemplates/chart/link-to-basics.md#gap-and-spacing)
 
+@[template](/_contentTemplates/chart/link-to-basics.md#configurable-nested-chart-settings)
+
+@[template](/_contentTemplates/chart/link-to-basics.md#configurable-nested-chart-settings-categorical)
+
+>caption Configuring Label Rotation, Skipping the rendering of every second label and adding borders and padding to the Labels.
+
+````CSHTML
+@* Skip rendering every second label and customize them to have borders, padding and rotation. *@
+
+<TelerikChart>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Column" Name="Product 1" Data="@series1Data">
+        </ChartSeries>
+        <ChartSeries Type="ChartSeriesType.Column" Name="Product 2" Data="@series2Data">
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis Categories="@xAxisItems">
+            <ChartCategoryAxisLabels Step="2"> @* With this you set the rendering step, in this case it will skip the rendering of every second label *@
+                <ChartCategoryAxisLabelsRotation Angle="-45"></ChartCategoryAxisLabelsRotation>
+                <ChartCategoryAxisLabelsPadding Top="10" Left="10" Right="10" Bottom="10"></ChartCategoryAxisLabelsPadding>
+                <ChartCategoryAxisLabelsBorder Width="1" Color="#FF0000" DashType="DashType.DashDot"></ChartCategoryAxisLabelsBorder>
+            </ChartCategoryAxisLabels>
+        </ChartCategoryAxis>
+    </ChartCategoryAxes>
+
+    <ChartTitle Text="Quarterly revenue per product"></ChartTitle>
+
+    <ChartLegend Position="ChartLegendPosition.Right">
+    </ChartLegend>
+</TelerikChart>
+
+@code {
+    public List<object> series1Data = new List<object>() { 10, 2, 5, 6, 8, 8, 13, 11, 4, 9, 10, 15, 14, 3, 2, 7 };
+    public List<object> series2Data = new List<object>() { 5, 8, 2, 7, 6, 11, 14, 13, 8, 7, 2, 7, 5, 9, 11, 10 };
+    public string[] xAxisItems = new string[] { "Q1", "Q2", "Q3", "Q4", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label", "loooooooong label" };
```

Comment:
Create data in a loop, you could use sequential numbers like "looong label N" so it is clearly visible what's skipped and rendered.